synapse/rest/admin/users.py

```diff
@@ -271,8 +299,35 @@ def __init__(self, hs):
 
                 current_time = self.hs.get_clock().time_msec()
                 for threepid in body["threepids"]:
+                    # For emails, transform the address to lowercase.
+                    # We store all email addreses as lowercase in the DB.
+                    # (See add_threepid in synapse/handlers/auth.py)
+                    address = threepid["address"].lower()
+                    if not check_3pid_valid_format(threepid["medium"], address):
+                        raise SynapseError(
+                            400,
+                            "Third party identifier has not a valid format",
```

Comment:
Please update each of these error messages to "Third party identifier has an invalid format".